poi/src/main/java/org/apache/poi/ss/util/SheetBuilder.java

```diff
@@ -117,7 +117,7 @@ public Sheet build() {
      * Sets the cell value using object type information.
      *
      * @param cell  cell to change
-     * @param value value to set
+     * @param value to set
```

Comment:
actually, i'm going to drop this. I have a partial rule to skip this, but apparently i wasn't careful enough...